lib/phoenix_mtm/helpers.ex

```diff
@@ -48,9 +106,10 @@ defmodule PhoenixMTM.Helpers do
         |> Keyword.put(:value, "#{value}")
         |> put_selected(selected, value)
 
-      input_tag = tag(:input, input_opts)
       label_opts = label_opts ++ [for: id]
-      build_label_with_input(form, field, input_tag, label, label_opts, opts)
+
+      mapper.(form, field, input_opts, label_text, label_opts, opts)
+      |> wrapper.()
```

Comment:
This is pretty cool, allowing both mapping and wrapping to take place at the same time, should it be needed!